provider/sriov/sriov_base.py

```diff
@@ -258,12 +258,11 @@ def get_rom_file(self):
         rom_vendor_device = lspci_stdout[1:-1].replace(':', '') + '.rom'
         rom_file = os.path.join('/usr/share/ipxe', rom_vendor_device)
         if not os.path.exists(rom_file):
-            build_cmd = "git clone https://github.com/ipxe/ipxe.git;\
-                         pushd ipxe/src; make bin/{0}; cp bin/{0} {1}; popd; \
-                         rm -rf ipxe".format(rom_vendor_device, rom_file)
-            process.run(build_cmd, shell=True, verbose=True)
-            if not os.path.exists(rom_file):
-                self.test.error("This test needs rom file: %s." % rom_file)
+            virtio_rom_file = "/usr/share/ipxe/1af41000.rom"
+            if not os.path.exists(virtio_rom_file):
+                self.test.error("This test needs a rom file, but neither {rom_file}"
+                                "nor {virtio_rom_file} exist" % rom_file)
```

Comment:
Plz use one format: 
```suggestion
                self.test.error(f"This test needs a rom file, but neither {rom_file}"
                                "nor {virtio_rom_file} exist")
```

OR
```suggestion
                self.test.error("This test needs a rom file, but neither %s"
                                "nor %s exist" % (rom_file, virtio_rom_file))
```

Reply:
Sorry, forgot to remove the previous `%s rom_file`. Just finish to update now.